README.md

```diff
@@ -75,6 +77,12 @@ Second, use the `pip` command from the previous step to install the package in t
 1. Make sure you can build the project by doing all of the `pip` steps above
 2. In PyCharm, run the _Unit Tests_ configuration.
 
+## Known Issues
++ Currently Parquet does not support TIMESTAMP_TZ and TIMESTAMP_LTZ data types so they are converted to TIMESTAMP_NTZ.
+  As a result timezone details are removed from the TIMESTAMP. See [Date & Time Data Types in Snowflake](https://docs.snowflake.com/en/sql-reference/data-types-datetime.html) for more details.
++ All TIMESTAMP (LTZ, TZ, NTZ) and DATE columns from Snowflake are stored as INTEGER in Parquet.
```

Comment:
```suggestion
* Snowflake stores all `TIMESTAMP_NTZ` and `DATE` columns as annotated logical types in Parquet. Because Dataiku does not support logical types, these appear in Dataiku as `bigint` (`int64` as `datetime`) and `int` (`int32` as `date`). More details and workarounds are described in #12 
```